components/coachmark/stories/template.js

```diff
@@ -1,4 +1,4 @@
-import { Template as ActionMenu } from "@spectrum-css/actionmenu/stories/template.js";
+import { Template as ActionButton } from "@spectrum-css/actionbutton/stories/template.js";
```

Comment:
Does changing this from action menu to just an action button imply anything for SWC? I only ask because the Figma specs for coachmark do specify coachmark being composed with a menu. I don't think it does, but this markup change will be noted in the component analysis, so we'll just have to be sure that we review those docs and clarify that the SWC component as is should still use an action menu.

<img width="213" height="227" alt="Screenshot 2025-10-17 at 1 24 52 PM" src="https://github.com/user-attachments/assets/97cb7be7-a485-4947-a76c-87465b100d4a" />

I also could be thinking too hard about it all.

Reply:
Yeah, I just removed the full implementation here for demonstration purposes but it doesn't change the end-result for SWC.